.github/workflows/main.yaml

```diff
@@ -23,6 +23,8 @@ jobs:
           yarn prettier --check .
           yarn eslint .
           yarn test
+          yarn validate-types-dev
+          yarn validate-types-prod
```

Comment:
These probably shouldn't be in the main flow, because we don't want deployment to the dev environment to fail if the production types don't match